crates/move-stackless-bytecode/src/stackless_bytecode.rs

```diff
@@ -208,6 +253,9 @@ pub enum Operation {
     // Event
     EmitEvent,
     EventStoreDiverge,
+
+    // Quantifiers
+    Quantifier(QuantifierType)
```

Comment:
this should contain both the quantifier type (`forall`, `map`, ...) and the function (`x_is_10`, `x_plus_10`, ...), so the function can be applied in boogie